srcpkgs/sayonara-player/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'sayonara-player'
+pkgname=sayonara-player
+_ver=1.6.0
+_rev=beta7
```

Comment:
> They seem to have some stable releases. Why go with beta here?

Because the latest stable release is 1.5 years old: https://gitlab.com/luciocarreras/sayonara-player/-/tags/1.5.1-stable5
BTW, `luajit` version is also beta: https://github.com/void-linux/void-packages/blob/master/srcpkgs/LuaJIT/template